examples/plot_clickable_image.py

```diff
@@ -0,0 +1,67 @@
+"""
+=========================================
+Demonstration of how to use ClickableImage / generate_2d_layout.
+=========================================
+
+In this example, we open an image file, then use ClickableImage to
+return 2D locations of mouse clicks (or load a file already created).
+Then, we use generate_2d_layout to turn those xy positions into a layout
+for use with plotting topo maps. In this way, you can take arbitrary xy
+positions and turn them into a plottable layout.
+"""
+
+from scipy.ndimage import imread
+import numpy as np
+from matplotlib import pyplot as plt
+import mne
+from mne.viz.utils import ClickableImage
+from mne.channels.layout import generate_2d_layout
+from os import path as op
+
+# Set parameters and paths
+plt.rcParams['image.cmap'] = 'gray'
+im_path = op.join(op.dirname(__file__), '..', 'mne', 'data', 'image', 'mni_brain.gif')
+
+# We've already clicked and exported
+layout_path = op.join(op.dirname(__file__), '..', 'mne', 'data', 'image', '')
+layout_name = 'custom_layout.lout'
+
+# Load data and click
+im = imread(im_path)
+
+"""
+This code opens the image so you can click on it. Commented out
+because we've stored the clicks as a layout file already.
+
+# The click coordinates are stored as a list of tuples
+click = ClickableImage(im)
+click.plot_clicks()
+coords = click.coords
+
+# Generate a layout from our clicks and normalize by the image
+lt = generate_2d_layout(np.vstack(coords), bg_image=im)
+lt.save(layout_path)  # To save if we want
+"""
+# We've already got the layout, load it
+lt = mne.channels.read_layout(layout_name, path=layout_path, scale=False)
+
+# Create some fake data
+nchans = len(lt.pos)
+nepochs = 50
+sr = 1000
+nsec = 5
+events = np.arange(nepochs).reshape([-1, 1])
+events = np.hstack([events, np.zeros([nepochs, 2])])
+data = np.random.randn(nepochs, nchans, sr * nsec)
+info = mne.create_info(nchans, sr, ch_types='eeg')
+epochs = mne.EpochsArray(data, info, events)
+
+# Using the native plot_topo function
+f = mne.viz.plot_topo(epochs.average(), layout=lt)
```

Comment:
Maybe instead of showing it with and without, instead show the image by itself up by the commented-out code (without calling `show`) and then just show the full thing, as you currently do, below. It's easy enough to imagine what the traces would look like without a background image. But it would be nice to see the image without traces on it so people could see that it had "target" click locations on it.